setup.py

```diff
@@ -16,7 +16,7 @@
     'click>=6.7',
     'deprecation>=2.1.0',
     'more-itertools>=8.6.0',
-    'morphio>=2.7.0',
+    'morphio>=2.7.0,<3',
     'neurom>=1.8.0',
```

Comment:
NeuroM v2 will require MorphIO > 3. That could conflict here in future.

Reply:
The test `test_diff.test_equality` fails with morphio==3. I can updated it to make it work with all versions.

Reply:
Does morph-tool require NeuroM v2 or v1?

Reply:
I just tested with `NeuroM==2.0.1.dev8` and the test `test_nrnhines.test_NeuroM_section_to_NRN_compartment_paths` fails. But since NeuroM v2 is not released yet we could fix this in another PR?

Reply:
yes, I'll make another PR to upgrade to morphio v3